server/src/main/java/org/opensearch/action/support/clustermanager/ClusterManagerNodeReadRequest.java

```diff
@@ -46,18 +47,25 @@
     ClusterManagerNodeRequest<Request> {
 
     protected boolean local = false;
+    protected boolean ensureLocalNodeCommissioned = false;
 
     protected ClusterManagerNodeReadRequest() {}
 
     protected ClusterManagerNodeReadRequest(StreamInput in) throws IOException {
         super(in);
         local = in.readBoolean();
+        if (in.getVersion().onOrAfter(Version.CURRENT)) {
+            ensureLocalNodeCommissioned = in.readBoolean();
+        }
     }
 
     @Override
     public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         out.writeBoolean(local);
+        if (out.getVersion().onOrAfter(Version.CURRENT)) {
+            out.writeBoolean(ensureLocalNodeCommissioned);
+        }
```

Comment:
Build fails in mixed cluster test if version check is PUT for 2.6 and not current. Please suggest correct way of doing this